test/powershell/engine/Job/Jobs.Tests.ps1

```diff
@@ -311,8 +311,17 @@ Describe 'Basic Job Tests' -Tags 'Feature' {
                 @{ property = 'InstanceId'}
                 @{ property = 'State'}
             )
-            # '-Seconds 100' is chosen to be substantially large, so that the job is in running state when Stop-Job is called.
-            $jobToStop = Start-Job -ScriptBlock { Start-Sleep -Seconds 100 } -Name 'JobToStop'
+            # 20 seconds is chosen to be large, so that the job is in running state when Stop-Job is called.
+            $jobToStop = Start-Job -ScriptBlock {
```

Comment:
There's 4 test cases, but BeforeAll only runs once.  Should you start the test job in BeforeEach?

Reply:
Yes, good point.  I'll also add another condition to the running wait loop.